specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/applications.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
1. All top-level ResourceTypes must have list operations on RG and Subscription scopes.
[https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/openapi-authoring-automated-guidelines.md#r4016-toplevelresourceslistbyresourcegroup](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/openapi-authoring-automated-guidelines.md#r4016-toplevelresourceslistbyresourcegroup)

Please refer to R4016 and R4017

Reply:
Added list and patch operations.